src/sqlfluff/utils/reflow/reindent.py

```diff
@@ -26,7 +26,8 @@
 )
 from sqlfluff.core.parser.segments import Indent, SourceFix
 from sqlfluff.core.parser.segments.meta import MetaSegment, TemplateSegment
-from sqlfluff.core.rules.base import LintFix, LintResult
+from sqlfluff.core.rules import LintFix
+from sqlfluff.core.rules.base import LintResult
```

Comment:
```suggestion
from sqlfluff.core.rules import LintFix, LintResult
```

Same comment as above...